cmd/mount_unix.go

```diff
@@ -153,7 +154,21 @@ func mount_flags() []cli.Flag {
 
 func disableUpdatedb() {
 	path := "/etc/updatedb.conf"
-	data, err := os.ReadFile(path)
+	file, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer file.Close()
+
+	// obtain exclusive and not block flock
+	if err := syscall.Flock(int(file.Fd()), syscall.LOCK_EX|syscall.LOCK_NB); err != nil {
+		return
+	}
+	defer func() {
+		// release flock
+		_ = syscall.Flock(int(file.Fd()), syscall.LOCK_UN)
```

Comment:
flock should be released when the file is closed

Reply:
syscall.LOCK_UN will release flock